demos/select.html

```diff
@@ -140,17 +134,21 @@ <h2 class="mdc-typography--title">Fully-Featured Component</h2>
           <label for="disabled">Disabled</label>
         </div>
       </section>
+
       <section class="example">
         <h2 class="mdc-typography--title">CSS Only</h2>
-        <select class="mdc-select">
-          <option value="" selected>Pick a food group</option>
-          <option value="grains">Bread, Cereal, Rice, and Pasta</option>
-          <option value="vegetables" disabled>Vegetables</option>
-          <option value="fruit">Fruit</option>
-          <option value="dairy">Milk, Yogurt, and Cheese</option>
-          <option value="meat">Meat, Poultry, Fish, Dry Beans, Eggs, and Nuts</option>
-          <option value="fats">Fats, Oils, and Sweets</option>
-        </select>
+        <div class="mdc-select">
+          <select class="mdc-select__surface">
+            <option value="" default selected>Pick a food group</option>
```

Comment:
I don't think `default` is a valid attribute for `<option>` ([MDN attribute reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes))

Reply:
Whoa. Yep.